lib/chat_web/live/main_live/page/feed.ex

```diff
@@ -124,4 +142,34 @@ defmodule ChatWeb.MainLive.Page.Feed do
       load_more(rest_count, small_list ++ list, till - 1)
     end
   end
+
+  defp assign_feed_stream(%{assigns: %{streams: %{action_feed_list: _feed}}} = socket, list) do
+    socket
+    |> stream_batch_insert(:action_feed_list, list,
+      at: -1,
+      dom_id: &"item-#{elem(&1, 0)}"
+    )
+    |> assign_items_uuid(list)
+  end
+
+  defp assign_feed_stream(socket, list) do
+    socket
+    |> stream(:action_feed_list, list, dom_id: &"item-#{elem(&1, 0)}")
```

Comment:
it should be at least `feed-item`, item is too general

Reply:
i would create a function to convert item to `dom_id`

Reply:
name changed and use new function `item_dom_id/1`